docs/zh/deploy/install_deploy.md

```diff
@@ -247,9 +193,6 @@ node3:7527
 [apiserver]
 node3:9080
 
-[taskmanager]
-localhost:9902
```

Comment:
dose not merge the latest upstream ?

Reply:
this should change to `node3:9902`

`localhost` is not accessible outside its own node

Reply:
@zhanghaohit pls double check